recipe/meta.yaml

```diff
@@ -12,13 +12,14 @@ source:
 build:
   number: 0
   script:
-    - pip install --no-deps .
+    - python setup.py install
     - python -m ipykernel install --prefix $PREFIX  # [not win]
     - python -m ipykernel install --prefix %PREFIX%  # [win]
 
 requirements:
   build:
     - python
+    - setuptools
```

Comment:
Done in https://github.com/conda-forge/pygments-feedstock/pull/4

I will remove `setuptools` from this as soon as I merge the fix in `pygments`.